sources/packages/backend/apps/queue-consumers/src/processors/schedulers/base-scheduler.ts

```diff
@@ -48,6 +51,17 @@ export abstract class BaseScheduler<T> implements OnApplicationBootstrap {
    * any old cron job delete it and add the new job to the queue.
    */
   async onApplicationBootstrap(): Promise<void> {
+    // Stops schedulers from running if their isActive status is false
+    // and they fall into the scheduler-queues category.
+    const inactiveQueueNames =
+      await this.queueService.getInactiveQueueConfigurations();
```

Comment:
Can we have `isActive` as an optional parameter in the existing method `getAllQueueConfigurations`?